inlang/source-code/sdk/src/v2/mocks/messageBundles.ts

```diff
@@ -0,0 +1,152 @@
+import type { MessageBundle } from "../types.js"
+
+export const examplePluralFormal: MessageBundle = {
+	id: "mock-bundle-human-id",
+	alias: {
+		default: "mock-bundle-alias",
+	},
+	messages: [
+		{
+			locale: "en",
+			declarations: [
+				{
+					type: "input",
+					name: "username",
+					value: {
+						type: "expression",
+						arg: {
+							type: "variable",
+							name: "username",
+						},
+					},
+				},
+			],
+			selectors: [
+				{
+					type: "expression",
+					arg: {
+						type: "variable",
+						name: "count: plural",
+					},
+				},
+			],
+			variants: [
+				{
+					match: ["one"],
+					pattern: [
+						{
+							type: "text",
+							value: "Show {count} message.",
```

Comment:
The `value`: `"Show {count} message."` doesn't treat "{count}" as a variable, but as plain text. 

The way to mix in variables into the AST is by adding `expression`s to the pattern.

```ts
pattern: [
  { type: "text", value: "Show "},
  { type: "expression", arg: { type: "variable", name: "count" }},
  { type: "text", value: " message."}
]
```

Reply:
+1

Reply:
Same ⤴️